temoto_context_manager/include/temoto_context_manager/env_model_repository.h

```diff
@@ -0,0 +1,199 @@
+#ifndef ENV_MODEL_REPOSITORY_H
+#define ENV_MODEL_REPOSITORY_H
+
+namespace temoto_context_manager 
+{
+namespace emr 
+{
+
+
+
+/**
+ * @brief Abstract base class for payloads
+ * 
+ */
+class PayloadEntry
+{
+private:
+  
+public:
+  std::string type;
```

Comment:
Since you already have a public getter function for the "type", is there a reason it could not be under private?

Reply:
Values were being assigned to it directly, I have now made a complementing setter and moved it to private.